src/ol/format/wfsformat.js

```diff
@@ -125,6 +125,9 @@ ol.format.WFS.prototype.readFeaturesFromNode = function(node, opt_options) {
   goog.object.extend(context, this.getReadOptions(node,
       goog.isDef(opt_options) ? opt_options : {}));
   var objectStack = [context];
+  this.gmlFormat_.FEATURE_COLLECTION_PARSERS[ol.format.GMLBase.GMLNS][
+      'featureMember'] =
+      ol.xml.makeArrayPusher(ol.format.GMLBase.prototype.readFeaturesInternal);
```

Comment:
It seems to me that, again, you're changing the `FEATURE_COLLECTION_PARSERS` object here, which is set on the `prototype` and therefore shared by all the GML formats one might have.

Is it intended?